app/routes/items/new.js

```diff
@@ -135,5 +152,6 @@ export default AuthorizeRoute.extend({
       }
       window.localStorage.setItem("isSelectLocationPreviousRoute", false);
     }
+    this.setupPrinterId(controller);
```

Comment:
Just for future reference. No need to pass `controller` here. Controller is available throughout route file.